apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap

```diff
@@ -5452,8 +5452,7 @@ snapshot_kind: text
         },
         "condition": {
           "$ref": "#/definitions/Condition_for_RouterSelector",
-          "description": "#/definitions/Condition_for_RouterSelector",
-          "nullable": true
```

Comment:
Since I removed the Option around the Selector, it now disallows explicit null values.
I'm not sure if it's something we should worry about?

If needed I can either write a config migration to add `null` or add support of explicit `null` to serde parsing.